cypress/integration/t-2.js

```diff
@@ -0,0 +1,149 @@
+describe('t-2 User role', function() {
+    
+    it('Should display validation for empty user after attempted loggin', function () {
+        cy.visit('/')
+        cy.get('.Select.not-valid').should('not.visible')
+        cy.get('[type="submit"]').click()
+        cy.get('.Select.not-valid').should('be.visible')
+    })
+
+    it('Should be able to login with role User', function () {
+        cy.visit('/')
+        cy.get('.Select.not-valid').should('not.visible')
+        cy.get('[type="submit"]').click()
+        cy.get('.Select.not-valid').should('be.visible')
+
+        cy.get('[id="loginForm.userId"]').click({force:true})
+        cy.get('[aria-label="TestCon User 22"]').click()
+        cy.get('[id="loginForm.role"]').click({force:true})
+        cy.get('[aria-label="User"]').click()
+        cy.get('[type="submit"]').click()
+
+        cy.url().should('include', '/time-logging')
+        cy.get('.page__title').contains('Timesheets')
+        cy.get('.calendar').should('be.visible')
+        cy.get('.tile.form').should('be.visible')
+        cy.get('.user-info__title').contains('TestCon User 22')
+        cy.get('.main-nav').find('li').should('have.length', 1)
+    })
+
+    it('check time logging tab', function () {
+        cy.get('.main-nav').find('li').find('a').should('have.class', 'main-nav__link--active').should('have.css', 'color', 'rgb(64, 76, 237)')
+    })
+})
+
+describe('t-2 Team lead role', function() {
```

Comment:
Hey there :) There's probably no need to extract tests for each role into separate describe blocks. I'd suggest to put them all under one single describe block called "T-2 All roles" or something, because all the tests are meant to test the same thing only with different roles.